default/templates/components/stork/stork-search-js.tpl

```diff
@@ -0,0 +1,45 @@
+<script src="https://files.stork-search.net/releases/v1.5.0/stork.js"></script>
+<ema:metadata>
+	<with var="template">
+		<script data-emanote-base-url="${value:baseUrl}">
+			window.searchShown = false;
+			function toggleSearch() {
```

Comment:
I'd like to see functions like `toggleSearch` not be made top-level; but rather inside a set, like `Emanote.toggleSearch`. Because then it won't have to conflict with user-defined top-level functions.